ideascube/serveradmin/tests/test_backup.py

```diff
@@ -32,61 +31,29 @@ def test_guess_format_from_name(input, expected):
     assert Backup.guess_file_format(input) == expected
 
 
+@pytest.mark.usefixtures('backup_root')
 def test_list(monkeypatch, settings):
     monkeypatch.setattr('ideascube.serveradmin.backup.Backup.ROOT',
                         BACKUPS_ROOT)
     settings.DATETIME_FORMAT = 'Y'
-    filename = 'edoardo-0.0.0-201501231405.zip'
+    filename = 'edoardo-0.0.0-201501231405.tar'
     good = os.path.join(BACKUPS_ROOT, filename)
-    bad = os.path.join(BACKUPS_ROOT, 'donotlistme.zip')
-    try:
-        os.makedirs(BACKUPS_ROOT)
-    except OSError:
-        pass  # Already exists.
+    bad = os.path.join(BACKUPS_ROOT, 'donotlistme.tar')
     open(good, mode='w')
     open(bad, mode='w')
     backups = list(Backup.list())
     assert len(backups) == 1
     assert backups[0].name == filename
     os.remove(good)
     os.remove(bad)
-    os.rmdir(BACKUPS_ROOT)
-
-
-def test_create_zipfile(monkeypatch, settings):
```

Comment:
Should you not replace that by a test that ensures we can't create zip files any more?